source/common/less/theme-berlin/light.less

```
@@ -211,6 +211,22 @@ body {
         border-bottom-right-radius: 4px;
     }
 
+    .code-block-copy-button{
+      position: absolute;
+      top: 1px;
+      right: 1px;
+      height: 20px;
+      width: 20px;
+      z-index: 8;
```

We should make the font a little bit smaller:

```suggestion
      z-index: 8;
      font-size: xx-small;
```